tinytroupe/utils/__init__.py

```
@@ -0,0 +1,17 @@
+"""
+General utilities and convenience functions.
+"""
+
+import logging
+logger = logging.getLogger("tinytroupe")
+
+###########################################################################
+# Exposed API
+###########################################################################
+from tinytroupe.utils.config import *
```

Ruff (0.8.2)

`from tinytroupe.utils.config import *` used; unable to detect undefined names

(F403)